server/handlers/cdn-caching.js

```diff
@@ -2,15 +2,26 @@ const _ = require("lodash");
 
 exports.addCacheHeadersToResult = function addCacheHeadersToResult(res, cacheKeys) {
   if(cacheKeys) {
-    res.setHeader('Cache-Control', "public,max-age=15,s-maxage=240,stale-while-revalidate=300,stale-if-error=14400");
-    res.setHeader('Vary', "Accept-Encoding");
+    if(cacheKeys === 'PRIVATE'){
+      res.setHeader('Cache-Control', "private,no-cache,no-store");
```

Comment:
add max-age=0

Reply:
Keep it in sync with surrogate-control

Reply:
Instead of "PRIVATE", let us use "DO_NOT_CACHE". I was explaining the PR to Asim, and it got confusing because "private" means something else wrt cache control headers.